.github/workflows/main.yml

```diff
@@ -44,5 +44,6 @@ jobs:
         with:
           node-version: 18
       - run: npm install
+      - run: npm install @types/node@12.0.0
```

Comment:
```suggestion
      - run: npm install @types/node@18.0.0
```

Reply:
I guess this part is no longer needed because of https://github.com/sindresorhus/type-fest/commit/8040abcbaae967797f11cb0dcbdb1aad6f3296a0

Reply:
sure